tests/test_smoketest.py

```diff
@@ -115,6 +141,28 @@ def test_smoketest(self):
             run(f"{venv_bin}hca dss get-subscriptions --replica {replica}")
             run(f"{venv_bin}hca dss delete-subscription --replica {replica} --uuid {sub_id}")
 
+        checkout_bucket = os.environ["DSS_S3_CHECKOUT_BUCKET"]
+        bundle_id = get_upload_val(".bundle_uuid")
```

Comment:
If you're going to load this into a variable, then please hoist it to immediately after bundle creation, and replace the shell subprocess calls (`$(jq -r .bundle_uuid upload.json)`) with format string substitutions (`{bundle_id}`).